cpp/Platform.Equality.Tests/Platform.Equality.Tests.cpp

```diff
@@ -0,0 +1,89 @@
+#include <Platform.Equality.h>
+#include <gtest/gtest.h>
+
+namespace Platform::Equality::Tests
+{
+    TEST(EqualityTest, BaseAnyEqual)
+    {
+        {
+            int a = 0x228;
+            int b = 0x1337;
+            ASSERT_NE(std::any(a), std::any(b));
+        }
+
+        {
+            int a = 0x177013;
+            float b = 0x177013;
+            ASSERT_EQ(std::any(a), std::any(b));
+        }
+
+        {
+            auto a = "i love green stuff";
+            auto b = (std::string)"i love green stuff";
+            ASSERT_NE(std::any(a), std::any(b));
+        }
+
+        {
+            auto a = "i love green stuff";
+            auto b = "i love green stuff";
+            ASSERT_EQ(std::any(a), std::any(b));
+        }
+
+        {
+            auto a = (std::string)"i love green stuff";
+            auto b = (std::string)"i love green stuff";
+            ASSERT_EQ(std::any(a), std::any(b));
+        }
+
+        {
+            std::vector<int> a {1, 2, 3, 4};
+            std::vector<int> b {1, 2, 3, 4};
+            //EXPECT_ANY_THROW(std::any(a) == std::any(b));
```

Comment:
Why did you comment this line out? The test should be working or should be removed.